src/main/MainTabs.js

```diff
@@ -11,11 +11,12 @@ import PmConversationsCard from '../pm-conversations/PmConversationsCard';
 import SettingsCard from '../settings/SettingsCard';
 import { IconHome, IconStream, IconSettings } from '../common/Icons';
 import IconUnreadConversations from '../nav/IconUnreadConversations';
+import EmojiPicker from '../emoji/EmojiPicker';
 
 export default TabNavigator(
   {
     home: {
-      screen: HomeTab,
+      screen: EmojiPicker,
```

Comment:
Try to do these types of 'debug convenience' changes in a separate commit, and just omit that commit from the PR. Makes it easier to review and makes the PR look closer to being done 👍